packages/web-components/src/components/image/image.scss

```diff
@@ -21,3 +23,13 @@
 :host(#{$dds-prefix}-image-item) {
   display: none;
 }
+
+:host(#{$dds-prefix}-card-cta-image) {
+  @include ratio-base(4, 3, true);
```

Comment:
Thanks Ari! One comment I have is if you put in a `2:1` image it makes things a little wonky. For some reason, this isn't a problem with the video thumbnail. I think you might be applying it to a specific component instead of the class name that they share.

![image](https://user-images.githubusercontent.com/3793636/111482464-ec0e2880-8701-11eb-9e5c-3297df667701.png)

Reply:
updated!